src/Components/MainTableView/TableContent.tsx

```diff
@@ -20,10 +21,10 @@ enum PaginationCommand {
 }
 
 enum TableActionType {
-  FILTER,
-  INSERT,
-  UPDATE,
-  DELETE
+  FILTER = 'Filter',
+  INSERT = 'Insert',
+  UPDATE = 'Update',
+  DELETE = 'Delete'
```

Comment:
Try to keep this as default autoincrement intergers cause that is vastly more efficeint then comparing strings.

If you want to get the strings, you can write a quick function to if and else the enum then return a string.

Reply:
addressed